topics/server.md

```diff
@@ -78,4 +78,4 @@ the config file:
 
 ## See also
 
-[Valkey documentation](./), [Introduction](introduction.md), [Configuration](valkey.conf.md), [Installation](installation.md), [valkey-cli](cli.md)
+[Valkey documentation](index.md), [Introduction](introduction.md), [Configuration](valkey.conf.md), [Installation](installation.md), [valkey-cli](cli.md)
```

Comment:
```suggestion
[Introduction](introduction.md), [Configuration](valkey.conf.md), [Installation](installation.md), [valkey-cli](cli.md)
```
Maybe a hot take, but you're either on the man pages or the website, you probably can already navigate back to the main doc index.